contracts/contracts/harvest/Harvester.sol

```diff
@@ -137,15 +157,16 @@ contract Harvester is Governable {
         );
         require(
             _uniswapV2CompatibleAddr != address(0),
```

Comment:
I would allow `_uniswapV2CompatibleAddr` to be zero address when the reward token is configured to be swapped using Uniswap V3.

A check that would make sense to me is `_uniswapV2CompatibleAddr ` != address(0) || _v3Fee > 0
meaning either UniswapV2 or Uniswap V3 should be configured. Maybe another check that both of them are not configured.